scripts/new_bare_repo

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+usage()
+{
+cat << EOF
+usage: new_bare_repo <hutch> <repo name>
+
+Script for automating the tedious process of integrating an existing IOC into our git bare repo scheme
+Run in the top level of your IOC
+
+Ex. new_bare_repo rix new_device
+(Creates a new bare repo in /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/rix/ named "new_device.git")
+
+You also might need to run kinit and aklog before hand!
+
+EOF
+}
+
+if [ $# -lt 1 ]; then
+   echo 'Need to specify the hutch and name of the proposed bare repo' >&2
+   usage
+   exit 1
+
+elif [[ ($1 == "--help") || ($1 == "-h") ]]; then
+   usage
+   exit 0
+fi
+
+HUTCH=$1
+REPO=$2
+
+if [ -d "/afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/" ] && [ ! -d "/afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/${REPO}.git" ]
+then
+    git init
+    cp /cds/sw/tools/bin/eco_tools/gitignore.template .gitignore
+    git add *.cfg Makefile .gitignore
+    git commit -m "Initial commit"
+    git-bare-repo /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/${REPO}.git
+    git remote add origin /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/${HUTCH}/${REPO}.git
+    git push -u origin master
```

Comment:
Out of scope: a future version of this script could help set up a github repo instead of an afs repo.
At some ill-defined future point we were going to integrate code review into our IOC process.

Reply:
Might be good to leave this `git push` step be manual, just in case the script does something you don't expect